external_deps/build.sh

```diff
@@ -521,6 +521,15 @@ build_naclsdk() {
 	cp pepper_*"/tools/irt_core_${NACLSDK_ARCH}.nexe" "${PREFIX}/irt_core-${DAEMON_ARCH}.nexe"
 	cp pepper_*"/toolchain/${NACLSDK_PLATFORM}_x86_newlib/bin/x86_64-nacl-gdb${EXE}" "${PREFIX}/nacl-gdb${EXE}"
 	rm -rf "${PREFIX}/pnacl"
+	
+	(
+		cd pepper_*"/toolchain/${NACLSDK_PLATFORM}_pnacl/bin/pydir"
+		if head -n 1 'pnacl-clang.py' | egrep -q '#!/usr/bin/python'
```

Comment:
Why is there an `if`? The patch must always be applied.

Reply:
Removed.